GameDataParser/Parsers/MagicPathParser.cs

```diff
@@ -53,8 +48,10 @@ protected override List<MagicPathMetadata> Parse()
                     bool ignoreAdjust = pathMove.Attributes?["ignoreAdjustCubePosition"] is null;
                     bool traceTargetOffsetPos = pathMove.Attributes?["traceTargetOffsetPos"]?.Value == "1";
                     float distance = float.Parse(pathMove.Attributes?["distance"]?.Value ?? "0");
+                    float velocity = float.Parse(pathMove.Attributes?["vel"]?.Value ?? "0");
+                    int lookAtType = int.Parse(pathMove.Attributes?["lookAtType"]?.Value ?? "0");
```

Comment:
Should we convert this to an enum? or too soon?

Reply:
Yeah in the long run lookAtType should be an enum, but I was going to hold off on that until I did a deeper dive into figuring out what exactly each value was used for. I don't want to create an enum for it when we don't have labels for any of the values yet.